sdk/servicebus/microsoft-azure-servicebus/src/test/java/com/microsoft/azure/servicebus/DaxiTest.java

```diff
@@ -0,0 +1,276 @@
+                                                   // receives message is passed to callback
+                                                   if (message.getLabel() != null &&
+                                                       message.getContentType() != null &&
+                                                       message.getLabel().contentEquals("Scientist") &&
+                                                       message.getContentType().contentEquals("application/json")) {
+
+                                                       byte[] body = message.getBody();
+                                                       Map scientist = GSON.fromJson(new String(body, UTF_8), Map.class);
+
+                                                       System.out.printf(
+                                                           "\n\t\t\t\tMessage received: \n\t\t\t\t\t\tMessageId = %s, \n\t\t\t\t\t\tSequenceNumber = %s, \n\t\t\t\t\t\tEnqueuedTimeUtc = %s," +
+                                                               "\n\t\t\t\t\t\tExpiresAtUtc = %s, \n\t\t\t\t\t\tContentType = \"%s\",  \n\t\t\t\t\t\tContent: [ firstName = %s, name = %s ]\n",
+                                                           message.getMessageId(),
+                                                           message.getSequenceNumber(),
+                                                           message.getEnqueuedTimeUtc(),
+                                                           message.getExpiresAtUtc(),
+                                                           message.getContentType(),
+                                                           scientist != null ? scientist.get("firstName") : "",
+                                                           scientist != null ? scientist.get("name") : "");
+                                                   }
+                                                   return CompletableFuture.completedFuture(null);
+                                               }
+
+                                               // callback invoked when the message handler has an exception to report
+                                               public void notifyException(Throwable throwable, ExceptionPhase exceptionPhase) {
+                                                   System.out.printf(exceptionPhase + "-" + throwable.getMessage());
+                                               }
+                                           },
+            // 1 concurrent call, messages are auto-completed, auto-renew duration
+            new MessageHandlerOptions(1, true, Duration.ofMinutes(1)),
+            executorService);
+
+    }
+
+    private void waitForEnter(int seconds) {
+        ExecutorService executor = Executors.newCachedThreadPool();
+        try {
+            executor.invokeAny(Arrays.asList(() -> {
+                System.in.read();
```

Comment:
System.in.read()? Will this block forever? Or do you expect users to interact with this test.